### ISSUE TITLE
[FEATURE]: Support env variable to avoid running `garden util hide-warning web-app`

### ISSUE THREAD

**Issue by bxsx** [garden-io/garden#6303]
### Background / Motivation

My organization integrates Garden into CI pipeline, running it multiple times across various jobs. Since this workflow does not utilize the Garden Dashboard, it generates confusing output for our users:

```
09:18:39 ℹ garden-dashboard          → You are not logged in. To use the Garden dashboard, log in with the garden login command.
09:18:39 ℹ garden-dashboard          → Learn more at: https://docs.garden.io/using-garden/dashboard
09:18:39 
🌿 Log in with garden login to explore logs, past commands, and your dependency graph in the Garden dashboard.
Learn more at: https://docs.garden.io/using-garden/dashboard

→ Run garden util hide-warning web-app to disable this message.
```

Running `garden util hide-warning web-app` takes approximately 4 seconds, multiplied by the number of jobs where Garden is used. This accumulates to a significant time cost.

### What should the user be able to do?

Setting up an environment variable equivalent to running the command above would be beneficial. Alternatively, it would be even better to completely disable the garden-dashboard component.

### Why do they want to do this? What problem does it solve?

Execution time in our CI pipeline and developer experience by reducing confusing output.

### Suggested Implementation(s)

Add the `GARDEN_DISABLE_WEB_APP` or `GARDEN_DISABLE_DASHBOARD` environment variable to hide the web-app warning.

If possible, completely disable the garden-dashboard component, including the output: 

```
05:50:26 ℹ garden-dashboard          → You are not logged in. To use the Garden dashboard, log in with the garden login command.
05:50:26 ℹ garden-dashboard          → Learn more at: https://docs.garden.io/using-garden/dashboard
```

### How important is this feature for you/your team?

Something between:
🌵 Not having this feature makes using Garden painful
🌹 It’s a nice to have, but nice things are nice 🙂

**Comment by vvagaytsev** [comment]
@bxsx we have implemented the flag called `GARDEN_DISABLE_WEB_APP_WARN`, it will be available in 0.13.36.

**Comment by bxsx** [comment]
Thanks @vvagaytsev appreciate it :)
